app/src/main/java/com/endava/myendava/fragments/ProfileFragment.java

```diff
@@ -173,12 +156,6 @@ private String getCurrentEmail() {
     @Override
     public void onResume() {
```

Comment:
onResume is empty

Reply:
In OnResume was the action on the both Floating and Export PDF buttons; mFloatingActionButton.setOnClickListener(view1 -> {
            mSharedPreferences.setUpNavigationId(R.id.navigation_profile);
            Intent intent = new Intent(ProfileFragment.this.getContext(), FilteredTagsActivity.class);
            startActivity(intent);
        });

       mExportButton.setOnClickListener(view12 -> Toast.makeText(getContext(), "Exported to PDF. Check your email.", Toast.LENGTH_SHORT).show());